runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -4140,6 +4140,32 @@ class INTERPRETER_CLASS
 
 		return rc;
 	}
+
+	VMINLINE VM_BytecodeAction
+	inlUnsafeGetObjectSize(REGISTER_ARGS_LIST)
+	{
+		j9object_t o = *(j9object_t*)_sp;
```

Comment:
instead of `o` you can use `receiver` which is a more descriptive name

Reply:
addressed in 99c66f9